ports/zydis/portfile.cmake

```
@@ -20,15 +23,17 @@ vcpkg_cmake_configure(
         -DZYDIS_BUILD_DOXYGEN=OFF
         -DZYDIS_BUILD_EXAMPLES=OFF
         -DZYDIS_BUILD_TESTS=OFF
-    OPTIONS_DEBUG
-        -DZYDIS_BUILD_TOOLS=OFF
+    OPTIONS_RELEASE
+        ${FEATURE_OPTIONS}
```

I still recommend:
```suggestion
        ${FEATURE_OPTIONS}
    OPTIONS_DEBUG
        -DZYDIS_BUILD_TOOLS=OFF
```
Overriding per config works reliably.
But expecting future contributors to see taht `FEAUTURE_OPTIONS` only affects releases is unreliable.